backend/upload.js

```diff
@@ -0,0 +1,24 @@
+import multer from 'multer'
+
+var Storage = multer.diskStorage({
+  destination: function (req, file, callback) {
+    callback(null, './images')
+  },
+  filename: function (req, file, callback) {
+    callback(null, file.fieldname + '_' + Date.now() + '_' + file.originalname)
+  }
+})
+
+var upload = multer({
+  storage: Storage,
+  limits: {fileSize: 1000000, files: 1} // Max 1 MB
```

Comment:
1MB is likely too low a limit. 50MB feels like a good number to me since highly-edited, hi-res images can be quite large. We should talk to Nanette about typical past file sizes.